.github/workflows/trigger-breaking-change-alert.yaml

```diff
@@ -0,0 +1,26 @@
+name: Trigger Breaking Change Notifications
+
+on:
+  pull_request_target:
+    types:
+      - closed
+      - reopened
+      - labeled
+      - unlabeled
+
+jobs:
+  trigger-notifier:
+    if: contains(github.event.pull_request.labels.*.name, 'breaking')
```

Comment:
Won't this also match `non-breaking`?

Reply:
Nope. I already tested for that to be sure (using [this PR](https://github.com/rapidsai/literate-octo-potato/pull/869)).

I think `github.event.pull_request.labels.*.name` maybe returns an iterable-like structure instead of a string.

Reply:
Nope. I already tested for that to be sure (using [this PR](https://github.com/rapidsai/literate-octo-potato/pull/869)).

I think `github.event.pull_request.labels.*.name` returns an iterable-like structure rather than a string.